lib/bulk_data_test_kit/v2.0.0/bulk_data_outputFormat_param_test.rb

```diff
@@ -21,6 +21,15 @@ class BulkDataOutputFormatParamTest < Inferno::Test
         as the abbreviated representations application/ndjson and ndjson.
       DESCRIPTION
 
+      verifies_requirements 'hl7.fhir.uv.bulkdata_2.0.0@47',
+                            'hl7.fhir.uv.bulkdata_2.0.0@49',
+                            'hl7.fhir.uv.bulkdata_2.0.0@51',
+                            'hl7.fhir.uv.bulkdata_2.0.0@52',
+                            'hl7.fhir.uv.bulkdata_2.0.0@230',
```

Comment:
it looks like the 230+ requirements are specific to output types: group / patient / system. Is there a place where these can be moved to?